install.sh

```diff
@@ -78,10 +115,17 @@ install_dependencies() {
             conntrack \
             firewalld
     else
-        sudo dnf install -y \
-            policycoreutils-python-utils \
-            conntrack \
-            firewalld
+        if [ "$DISTRO" = "centos" ] && [ "$VERSION" = "7" ]; then
```

Comment:
we aren't targetting centos 7, centos 7 uses iptables-legacy (instead of nft based) and adds a set of complications we don't have time to look at now.

Reply:
removed!